lib/libesp32/berry/src/be_lexer.c

```diff
@@ -269,6 +270,11 @@ static void skip_comment(blexer *lexer)
             }
             c = next(lexer);
         } while (!(mark && c == '#') && c != EOS);
+        if (c == EOS) {
+        	char tmp[64];
```

Comment:
Ouch sorry. I didn't check the details. You shouldn't be using sprintf as it is considered as a dangerous function. 

You should be doing `be_lexererror(lexer, be_pushfstring(lexer->vm,...))`

Check in be_parser.c for examples (push_error)